spec/lib/authorities/linked_data/generic_authority_spec.rb

```diff
@@ -599,5 +114,51 @@
         expect(min_authority.term_id_expects_uri?).to eq true
       end
     end
+
+    describe '#search' do
+      it 'responds to delegated search method' do
+        expect(min_authority).to respond_to :search
+      end
+    end
+
+    describe '#find' do
+      it 'responds to delegated search method' do
```

Comment:
Added as a replacement for the moved find methods to ensure that find can be called from here still.